buildchain/buildchain/targets/package.py

```diff
@@ -375,7 +372,7 @@ def build_package(self) -> types.TaskDict:
                 source=self.debuild_sources, target=Path('/debbuild/pkg-meta')
             ),
             utils.bind_mount(
-                source=self.rootdir, target=Path('/debbuild/result')
+                source=constants.PKG_DEB_ROOT, target=Path('/debbuild/result')
```

Comment:
From my comment above, this one would change as well.

Reply:
It will be handled by the issue #1781